renovate.json

```
@@ -1,5 +1,16 @@
 {
   "extends": [
     "github>Kong/public-shared-renovate:kong-frontend-config.json"
+  ],
+  "packageRules": [
+    {
+      "matchPackageNames": [
+        "node"
+      ],
+      "matchDepTypes": [
+        "engines"
+      ],
+      "enabled": false
+    }
```

I forgot to give you some additional properties:
```suggestion
    {
      "groupName": "node engines range",
      "groupSlug": "node-engines",
      "matchPackageNames": [
        "node"
      ],
      "matchDepTypes": [
        "engines"
      ],
      "enabled": false
    }
```